mind_matter_api/api/users.py

```
@@ -0,0 +1,156 @@
+from flask import jsonify, request, session
```

Ruff (0.8.2)

`flask.session` imported but unused

Remove unused import: `flask.session`

(F401)